zuul.d/molecule.yaml

```diff
@@ -1,305 +1,304 @@
-# DO NOT EDIT - generaged using make role_molecule
 - job:
-    name: cifmw-molecule-artifacts
-    parent: cifmw-molecule-base
-    vars:
-      TEST_RUN: artifacts
     files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/artifacts/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
-- job:
-    name: cifmw-molecule-build_openstack_packages
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/polarion/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-polarion
     parent: cifmw-molecule-base
     vars:
-      TEST_RUN: build_openstack_packages
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/build_openstack_packages/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
+      TEST_RUN: polarion
 - job:
-    name: cifmw-molecule-ci_setup
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/os_must_gather/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-os_must_gather
     parent: cifmw-molecule-base
     vars:
-      TEST_RUN: ci_setup
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/ci_setup/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
+      TEST_RUN: os_must_gather
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/copy_container/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-copy_container
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: copy_container
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/copy_container/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
-    name: cifmw-molecule-discover_latest_image
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/set_openstack_containers/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-set_openstack_containers
```

Comment:
pretty sure that one also needs to get a crc-based nodeset, when reading the reported CI failure.

Reply:
Updated the parent based on @raukadah info https://github.com/openstack-k8s-operators/ci-framework/pull/401/files#diff-e11625dd628d0cfeeaad59207c1fabdd877fb86c87eeca532b83571e184628dcR17